integration-tests/deployment/ccip/changeset/initial_deploy_test.go

```diff
@@ -91,17 +97,126 @@ func TestInitialDeploy(t *testing.T) {
 	// Wait for all commit reports to land.
 	ccdeploy.ConfirmCommitForAllWithExpectedSeqNums(t, e, state, expectedSeqNum, startBlocks)
 
+	// Token and Gas prices should be updated in FeeQuoter
+	assertUpdatedGas(t, e, state, initialGasUpdates)
```

Comment:
Does the ConfirmCommitForAllWithExpected guarantee that the price reports would be included there? Or how do we know that the price reports will have landed by this point

Reply:
updating prices is part of the OffRamp commit function. so if commit happened it's safe to assume prices were updated. (unless there's a bug)